docs/0-intended-use/index.md

```diff
@@ -183,9 +203,10 @@ specific servers.
 
 ## Operating Principle
 
-<!-- > It's kind of a stretch to describe the "operating principle" of software. I guess this makes more sense for
-> hardware devices. In any case, I'd just generally state what sort of input goes in and what output comes
-> out, e.g. you could be processing images and returning diagnoses. -->
+<!-- > It's kind of a stretch to describe the "operating principle" of software. I
+> guess this makes more sense for hardware devices. In any case, I'd just
+> generally state what sort of input goes in and what output comes out, e.g.
+> you could be processing images and returning diagnoses. -->
```

Comment:
Couldn't quite comment on the specific line of interest, but this was closest.
You mention `standard DICOM input output (either file based or DICOM network protocol based)` within the operating principle
Technically, DICOMWeb doesn't fall in to either of those, so you might want to include it

Reply:
I'm actually not using DICOMWeb at the moment...

Reply:
pity

Reply:
Hehe :)

When I was using an online service I was actually utilising gRPC which was absolutely beautiful :slightly_smiling_face:. But, all reference in these docs to a cloud service will actually be going away initially, as my first version will be entirely offline.

Or, if there is an online section, it will be just for user account management and usage tracking for billing purposes.